docs/index.md

```diff
@@ -31,3 +31,22 @@ A modern Node.js framework based on koa@2 that comes powered with RabbitMQ, Kafk
 
 [Console](https://workable.github.io/orka/integrations/console)
 
+
+## Changelog
+
+### Migrating from orka  1.x.x to 2.xx
```

Comment:
@klesgidis most things affect you. Can you check?

Reply:
Yes, we need some changes here, before release. 
cc: @johnvlach

Reply:
Created https://workable.atlassian.net/browse/PROD-2136 for this purpose